crud/borders.lua

```diff
@@ -44,14 +45,25 @@ function borders.init()
    _G._crud.get_border_on_storage = get_border_on_storage
 end
 
-local function is_closer(compare_sign, keydef, tuple, res_tuple)
-    if res_tuple == nil then
-        return true
-    end
+local is_closer
+
+if has_keydef then
```

Comment:
I sure that it should be encapsulated in compare.keydef.I mean, using comparators instead of keydef.

Reply:
Oh, no

Reply:
It's just bugfix for annoying refactoring create a separate issue and take it to sprint.

Reply:
There is another solution: simply include tuple_keydef to sdk: https://github.com/tarantool/sdk/pull/210

Reply:
> There is another solution: simply include tuple_keydef to sdk: [tarantool/sdk#210](https://github.com/tarantool/sdk/pull/210)

It's not a solution at all. Because there are some Tarantool versions that does not support tuple-keydef/tuple-merger modules.

Reply:
> File an issue please.

https://github.com/tarantool/crud/issues/168